src/EditorFeatures/Core/Extensibility/Commands/ExportCommandHandlerAttribute.cs

```diff
@@ -8,13 +8,13 @@ namespace Microsoft.CodeAnalysis.Editor
 {
     [MetadataAttribute]
     [AttributeUsage(AttributeTargets.Class)]
-    internal class ExportCommandHandlerAttribute : ExportAttribute
+    internal class ExportLegacyCommandHandlerAttribute : ExportAttribute
```

Comment:
File wasn't renamed to match. I admit I'm also on the ExportRoslynCommandHandler side of things.